packages/ebayui-core-react/src/ebay-file-preview-card-group/types.ts

```diff
@@ -0,0 +1,30 @@
+import { EbayEventHandler } from '../common/event-utils/types'
+
+export type FilePreviewCardMenuAction = {
+    event: string
+    label: string
+}
+
+export type FilePreviewType = {
+    name: string
+    type?: File['type']
+    src?: string
+}
+
+type EventData = {
+    index: number
+}
+
+type MenuActionCardGroupEventData = {
+    cardIndex: number
+    index?: number
+    checked?: number[]
+    eventName?: string
+}
+
+export type FilePreviewCardActionHandler = EbayEventHandler<Element, EventData>
+
+export type FilePreviewCardMenuActionHandler = EbayEventHandler<
+    Element,
```

Comment:
Lets use `HTMLElement` to be the same as we use on the [preview-card component](https://github.com/eBay/evo-web/blob/main/packages/ebayui-core-react/src/ebay-file-preview-card/types.ts#L21C5-L21C16)